src/App/App.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import './App.css';
+import Header from '../Header/Header';
+import Footer from '../Footer/Footer';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+
+    }
+  }
+
+  render() {
```

Comment:
off to an awesome start with this App.js file. Glad you got everything hooked up and working!